src/main/java/com/gtnewhorizons/angelica/client/font/BatchingFontRenderer.java

```diff
@@ -462,13 +479,13 @@ public float drawString(final float anchorX, final float anchorY, final int colo
                     final int endColumn = this.glyphWidth[chr] & 15;
                     final float startColumnF = (float) startColumn;
                     final float endColumnF = (float) (endColumn + 1);
-                    uStart = ((float) (chr % 16 * 16) + startColumnF) / 256.0f;
-                    vStart = ((float) ((chr & 255) / 16 * 16)) / 256.0f;
+                    uStart = ((float) (chr % 16 * 16) + startColumnF + 0.21f) / 256.0f;
```

Comment:
Where did you get the 0.21f magic number from? It's slightly less than a quarter of a pixel, which seems weird